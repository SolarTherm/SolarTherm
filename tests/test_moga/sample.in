
# Dakota Input File: sample.in
# Usage:
#    dakota -i sample.in -o sample.out > sample.stdout
# or if run in multi-level parallelism:
#    mpirun dakota -i sample.in -o sample.out > sample.stdout

environment
    tabular_data
    tabular_data_file = "sample.dat"

model
    single

interface
    fork
    analysis_drivers = "/home/john/solartherm/tests/test_moga/interface_bb.py"
    parameters_file = "params.in"
    results_file = "results.out"
	file_tag 
	#file_save 

responses
	objective_functions = 2
no_gradients
no_hessians


variables
    discrete_state_set
        string 18
        set_values  "/home/john/solartherm/tests/TestStOptimise.mo"  "TEST"  "0"  "1"  "5m"  "None" "None" "5"  "dassl"  "homotopy"  "-LOG_SUCCESS,-stdout"  "False"  "False"  "2"  "f_schaffer1"  "f_schaffer2"  "1.0"  "1.0"
        descriptors  "fn"  "system"  "start"  "stop"  "step"  "initStep"  "maxStep"  "integOrder"  "solver"  "nls"  "lv"  "runsolstice" "peaker" "num_perf"  "index0"  "index1"  "sign0"  "sign1"
    continuous_design=1
        initial_point 0.1
        lower_bounds -1.0
        upper_bounds 1.0
        descriptors "x2"
	
method

    moga
        seed = 10983
    max_function_evaluations = 2500
    initialization_type unique_random
    crossover_type shuffle_random
        num_offspring = 2 num_parents = 2
        crossover_rate = 0.8
    mutation_type replace_uniform
        mutation_rate = 0.1
    fitness_type domination_count
    replacement_type below_limit = 6
        shrinkage_fraction = 0.9
    convergence_type metric_tracker
        percent_change = 0.05 num_generations = 40
    final_solutions = 3
    output silent

