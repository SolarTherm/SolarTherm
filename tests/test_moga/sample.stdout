8 total processes failed to start
